testsuite/integration-arquillian/tests/base/src/main/java/org/keycloak/testsuite/arquillian/AuthServerTestEnricher.java

```diff
@@ -164,16 +169,24 @@ public void initializeSuiteContext(@Observes(precedence = 2) BeforeSuite event)
                 throw new IllegalStateException("Frontend container (load balancer) misconfiguration");
             }
 
+            boolean authServerJboss = Boolean.parseBoolean(System.getProperty("auth.server.jboss.crossdc"));
+            String containerPreffix;
+            if (authServerJboss) {
+                containerPreffix = "auth-server-jboss-cross-dc-";
+            } else { //undertow
+                containerPreffix = AUTH_SERVER_CONTAINER + "-cross-dc-";
+            }
             containers.stream()
-              .filter(c -> c.getQualifier().startsWith(AUTH_SERVER_CONTAINER + "-cross-dc-"))
-              .sorted((a, b) -> a.getQualifier().compareTo(b.getQualifier()))
-              .forEach(c -> {
-                String portOffsetString = c.getArquillianContainer().getContainerConfiguration().getContainerProperties().getOrDefault("bindHttpPortOffset", "0");
-                String dcString = c.getArquillianContainer().getContainerConfiguration().getContainerProperties().getOrDefault("dataCenter", "0");
-                updateWithAuthServerInfo(c, Integer.valueOf(portOffsetString));
-                suiteContext.addAuthServerBackendsInfo(Integer.valueOf(dcString), c);
-              });
+                    .filter(c -> c.getQualifier().startsWith(containerPreffix))
+                    .sorted((a, b) -> a.getQualifier().compareTo(b.getQualifier()))
```

Comment:
Why so many changes in whitespace (in this hunk)?

Reply:
No specific reason. I've noticed some redundant white spaces, so I've removed them. If it is required I can revert those.